airbyte-lib/tests/integration_tests/test_source_test_fixture.py

```diff
@@ -27,23 +28,43 @@
 
 
 @pytest.fixture(scope="module", autouse=True)
-def prepare_test_env():
+def source_test_installation():
     """
-    Prepare test environment. This will pre-install the test source from the fixtures array and set the environment variable to use the local json file as registry.
+    Prepare test environment. This will pre-install the test source from the fixtures array and set
+    the environment variable to use the local json file as registry.
     """
     if os.path.exists(".venv-source-test"):
         shutil.rmtree(".venv-source-test")
 
     os.system("python -m venv .venv-source-test")
     os.system(".venv-source-test/bin/pip install -e ./tests/integration_tests/fixtures/source-test")
 
-    os.environ["AIRBYTE_LOCAL_REGISTRY"] = "./tests/integration_tests/fixtures/registry.json"
-    os.environ["DO_NOT_TRACK"] = "true"
-
```

Comment:
Moved to a function-scoped fixture and locally-scoped patch method to avoid bleeding 'AIRBYTE_LOCAL_REGISTRY' to other files.